### DISCUSSION TITLE
get tracking summary from package_search

### DISCUSSION THREAD

**Discussion by chonhenry** [ckan/ckan#7171] [Q&A]
is there any way to get the tracking summary from the package_search api?

**Comment by amercader** [comment]
The `tracking_summary` was removed from the indexed dataset_dict for performance reasons. There is work to bring it back in https://github.com/ckan/ckan/pull/5673, but in the meantime you will have to index it yourself in a `before_index()` plugin hook. 

If you are querying a CKAN instance that you don't control, right now there's no way to get the tracking info from package_search
